doc/src/installation.rst

```diff
@@ -0,0 +1,161 @@
+.. highlightlang:: none
+
+************
+Installation
+************
+
+Quick-start
+===========
+
+PyFR |release| can be installed using `pip <https://pypi.python.org/pypi/pip>`_
+and `virtualenv <https://pypi.python.org/pypi/virtualenv>`_, as shown in the
+quick-start guides below.
+
+Alternatively, PyFR |release| can be installed from 
+`source <http://www.pyfr.org/download.php>`_, see :ref:`compile-from-source`.
+
+macOS
+-----
+
+We recommend using the package manager `homebrew <https://brew.sh/>`_.
+Open the terminal and install the dependencies with the following commands::
+
+    brew install python3 open-mpi metis
+    pip3 install virtualenv
+
+For visualisation of results, either install Paraview from the command line::
+
+    brew cask install paraview
+
+or dowload the app from the Paraview `website <https://www.paraview.org/>`_.
+Then create a virtual environment and activate it::
+
+    virtualenv --python=python3 ENV3
+    source ENV3/bin/activate
+
+Finally install PyFR with `pip <https://pypi.python.org/pypi/pip>`_ in the
+virtual environment::
+
+    pip install pyfr
+
+This concludes the installation. In order to run PyFR with the OpenMP backend
+(see :ref:`running-pyfr`), use the following settings in the configuration file
```

Comment:
You could make the reference to the config file actually link to the relevant page with :ref:`...` ?